chore: Documentation devcontainer setup (#3943)
arcjet/arcjet-js: CONTRIBUTING.md

Please review the guidelines we have below to help us in accepting your
contribution.

# Setup

> We'll use Dev Containers to provide fully configured development enviroment.

Reviewer: ```suggestion
We recommend using [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers) to provide a fully configured development environment.
```

1. Prerequisites
  - Docker
  - VS Code or a compatible editor.
  - [Dev Container extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)

2. Open in Dev Container
 - Clone the repository
 - Open the project in VS Code.

>  - When prompted, click "Reopen in Container" or run "Dev Containers: Open Folder in Container"       command

Reviewer: ```suggestion
 - When prompted, click "Reopen in Container" or run "Dev Containers: Open Folder in Container" command
```
